src/sentry/static/sentry/app/views/releases/utils/sessionTerm.tsx

```diff
@@ -0,0 +1,113 @@
+import {PlatformKey} from 'app/data/platformCategories';
+import {t} from 'app/locale';
+
+const commonTermsDescription = {
+  crashes: t('Number of sessions with a crashed state'),
+  'crash-free-users': t('Number of unique users with non-crashed sessions'),
+  'crash-free-sessions': t('Number of non-crashed sessions'),
```

Comment:
Should this be a `number` or rather `percentage`?

Reply:
I will address this in a follow-up PR ;-)